_includes/sidebar.html

```diff
@@ -47,7 +48,7 @@
 
       <a class="sidebar-nav-item active" href="CppCoreGuidelines.html">C++ Core Guidelines</a>
       <a class="sidebar-nav-item active" href="index.html">README</a>
-      <a class="sidebar-nav-item active" href="CONTRIBUTING.html">Contributing</a>
+      <a class="sidebar-nav-item active" href="CONTRIBUTING.md">Contributing</a>
```

Comment:
In the gh-opages branch, there is an index.html which is loaded when you visit:
http://isocpp.github.io/CppCoreGuidelines

That one has a sidebar configured in _includes/sidebar which allows some navigation.

What I now used in a different project is a redirecting index.html, maybe that would be preferable for you:
```
<!DOCTYPE html>  
<html>  
<head>  
  <title>Redirecting to main site</title>
  <meta charset="UTF-8">
  <meta http-equiv="refresh" content="0;url=README.html">
</head>  
<body>  
  Redirecting<i>↻</i>
</body>  
</html>  
```

Then you could also get rid of that section in the sidebar.html, and put LICENSE and CONTRIBUTING into the ignore list in `_config.yaml`. I can make a PR for that if you want.